.github/workflows/push.yml

```diff
@@ -36,4 +36,4 @@ jobs:
         path: ./out
 
     - name: Publish to Azure Artifacts
-      run: dotnet nuget push ./out/*.nupkg --source https://www.myget.org/F/neo/api/v3/index.json --api-key ${MYGET_TOKEN} 
+      run: dotnet nuget push ./out/*.nupkg --source https://www.myget.org/F/neo/api/v3/index.json --api-key ${{ secrets.MYGET_TOKEN }}
```

Comment:
Do you still need token, or its already set?

Reply:
from the error is says it isnt set

Reply:
can you check

Reply:
@shargon can you help?